Dockerfile

```diff
@@ -22,10 +22,13 @@ RUN conda env create -n wps -f environment.yml
 # Install WPS
 RUN ["/bin/bash", "-c", "source activate wps && python setup.py develop"]
 
+# Init DB
+RUN ["/bin/bash", "-c", "source activate wps && pywps -c /opt/wps/pywps.cfg migrate"]
```

Comment:
Just curious, why we need this DB migrate/init step now and not before?  Isn't pywps always using a DB (sqlite or postgres) before?

Reply:
Before there were no updates to the pywps db schema. With the "watchdog" pywps we will have one and Jachym has added alembic to pywps which needs this migration step.

Reply:
Ah, understood, thanks.